vumi_twilio_api/tests/test_twilio_api.py

```diff
@@ -430,6 +445,21 @@ def parse_say(twiml):
         self.assertEqual(verb.text, 'foobar')
 
     @inlineCallbacks
+    def test_receive_call_parsing_play_verb(self):
+        response = twiml.Response()
+        response.play('test_url')
+        self.twiml_server.add_response('', response)
+
+        msg = self.app_helper.make_inbound(
+            '', from_addr='+54321', to_addr='+12345',
```

Comment:
Minor thought -- I think ``None`` is a more realistically content value for new sessions than ``''``.